testing/scenario_app/ios/Scenarios/ScenariosTests/FlutterViewControllerTest.m

```diff
@@ -25,25 +25,29 @@ - (void)tearDown {
 }
 
 - (void)testFirstFrameCallback {
+  XCTestExpectation* firstFrameRendered = [self expectationWithDescription:@"firstFrameRendered"];
+
   FlutterEngine* engine = [[FlutterEngine alloc] initWithName:@"test" project:nil];
   [engine runWithEntrypoint:nil];
   self.flutterViewController = [[FlutterViewController alloc] initWithEngine:engine
                                                                      nibName:nil
                                                                       bundle:nil];
-  __block BOOL shouldKeepRunning = YES;
+
+  XCTAssertFalse(self.flutterViewController.isRenderingFrames);
+
+  [self keyValueObservingExpectationForObject:self.flutterViewController
```

Comment:
I forget the exact API but there's another one that let's you set the expected value as a parameter to the expectation. `@YES` will probably work even though it's a primitive, though I might be wrong. Then you won't need the assert true at the end.

Reply:
Done